app/src/mutations/webhook.rs

```diff
@@ -126,6 +127,105 @@ impl Mutation {
+            .filter(webhooks::Column::Id.eq(input.webhook))
+            .one(db.get())
+            .await?
+            .ok_or_else(|| Error::new("webhook not found"))?;
+
+        let org_app = organization_applications::Entity::find()
+            .filter(
+                organization_applications::Column::OrganizationId
+                    .eq(webhook.organization_id.clone()),
+            )
+            .one(db.get())
+            .await?
+            .ok_or_else(|| Error::new("organization not found"))?;
+
+        let app_id = org_app.svix_app_id;
+
+        let update_endpoint = EndpointUpdate {
+            channels: Some(input.projects.iter().map(ToString::to_string).collect()),
+            filter_types: Some(input.filter_types.iter().map(|e| e.format()).collect()),
+            version: 1,
+            description: Some(input.description),
+            disabled: input.disabled,
+            rate_limit: input.rate_limit,
+            url: input.endpoint,
+            uid: Some(webhook.id.clone().to_string()),
+        };
+
+        let endpoint = svix
+            .endpoint()
+            .update(
+                app_id.clone(),
+                webhook.endpoint_id.clone(),
+                update_endpoint,
+                None,
+            )
+            .await?;
+
+        let current_projects: Vec<Uuid> = webhook_projects::Entity::find()
+            .filter(webhook_projects::Column::WebhookId.eq(webhook.id.clone()))
+            .select_only()
+            .column(webhook_projects::Column::ProjectId)
+            .all(db.get())
+            .await?
+            .into_iter()
+            .map(|p| p.project_id.clone())
+            .collect();
+
+        for project in input.projects.clone() {
+            if !current_projects.contains(&project) {
+                let webhook_project_active_model = webhook_projects::ActiveModel {
+                    webhook_id: Set(webhook.id),
+                    project_id: Set(project),
+                    ..Default::default()
+                };
+                webhook_project_active_model.insert(db.get()).await?;
+            }
+        }
+
+        for project in current_projects.clone() {
+            if !input.projects.contains(&project) {
+                let webhook_project_active_model = webhook_projects::ActiveModel {
+                    webhook_id: Set(webhook.id),
+                    project_id: Set(project),
+                    ..Default::default()
+                };
+                webhook_project_active_model.delete(db.get()).await?;
+            }
+        }
+
+        let webhook_active_model = webhooks::ActiveModel {
+            endpoint_id: Set(endpoint.id.clone()),
+            organization_id: Set(webhook.organization_id.clone()),
+            updated_at: Set(Some(Utc::now().naive_utc())),
+            created_by: Set(user_id),
+            ..Default::default()
+        };
+
+        let webhook = webhook_active_model.insert(db.get()).await?;
```

Comment:
The webhook already exists so you need to find the current one update the fields and do an update not insert.